lib/rules/declaration-block-properties-order/__tests__/validate-options.js

```diff
@@ -15,8 +21,10 @@ it("valid default order", () => {
     code: "",
     config,
   }).then(function (data) {
-    const invalidOptionWarnings = data.results[0].invalidOptionWarnings
-    expect(invalidOptionWarnings.length).toBe(0)
+    const results = data.results[0]
+    expect(results.deprecations.length).toEqual(1)
+    expect(results.deprecations).toEqual(deprecationResult)
+    expect(results.invalidOptionWarnings.length).toBe(0)
   })
```

Comment:
Any of the tests in this particular file that tested `declaration-block-properties-order` should just be removed, I think. We don't want `validate-options` tests to be tangled up with deprecations.